elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -435,6 +445,30 @@ bool ElevationMapping::getSubmap(grid_map_msgs::GetGridMap::Request& request, gr
   return isSuccess;
 }
 
+bool ElevationMapping::getRawSubmap(grid_map_msgs::GetGridMap::Request& request, grid_map_msgs::GetGridMap::Response& response)
+{
+  grid_map::Position requestedSubmapPosition(request.position_x, request.position_y);
+  Length requestedSubmapLength(request.length_x, request.length_y);
+  ROS_DEBUG("Elevation submap request: Position x=%f, y=%f, Length x=%f, y=%f.", requestedSubmapPosition.x(), requestedSubmapPosition.y(), requestedSubmapLength(0), requestedSubmapLength(1));
```

Comment:
I would add "raw" to the output, otherwise it is not possible to distinguish between the outputs of fused/raw methods.